### DISCUSSION TITLE
посмотреть / возможно сразу интегрировать Super Productivity

### DISCUSSION THREAD

**Discussion by Creatyclub** [Creaty-Co/creaty-api#243] [Ideas]

**Comment by Creatyclub** [comment]
@GentleEnvy 
напиши как разберешься по времени / вопросам, которые нужно уточнить

**Comment by GentleEnvy** [comment]
![image](https://github.com/Creaty-Co/creaty-api/assets/58393514/ecb12970-aa15-410c-a17d-585c34ab69a9)

Там нет автоматического трекинга времени, это простой таймер, где надо нажимать start / stop

**Comment by GentleEnvy** [comment]
Не будем
